chef/cookbooks/network/recipes/default.rb

```diff
@@ -314,9 +334,76 @@ def kill_nic(nic)
   if network.add_ovs_bridge
     bridge = network.bridge_name || "br-#{network.name}"
 
+    # Check if our current base interface (bond, vlan or physical) is already
+    # attached to another OVS bridge. Take care about that appropriately.
+    if our_iface.ovs_master && our_iface.ovs_master.name != bridge
+      old_ovs = our_iface.ovs_master
+
+      # We are already attached to an ovs bridge, check whether that attachment
+      # was already verified (or created) during this chef-client run, by looking
+      # through the list of interfaces that we already touched ('ifs'). If it is,
+      # find out which crowbar network requires that attachement and error out
+      # with a meaningful error message.
+      if ifs.key? old_ovs.name
+        other_net = sorted_networks.find do |n|
+          n.bridge_name == old_ovs.name
+        end
+        other_net ||= sorted_networks.find do |n|
+          n.name == old_ovs.name.sub(/\A#{Regexp.escape("br-")}/, "")
```

Comment:
Some explanation of this 'br-XXX' vs 'XXX' would be helpful.

Reply:
I'll do that. Though I assumed is was pretty obvious, that the bridge's name is derived from the network name in crowbar.

Reply:
OK, I misread the code above. I thought that the bridge could be named XXX or br-XXX. Still I think this code could be done simpler:
a. `"br-#{n.name}" == old_ovs.name`
b. `n.name == old_ovs.name.sub(/\Abr-/, "")` I don't think we need to escape `-` outside of character range

Reply:
Done